### PR TITLE
Features: Redeploys

### PR DESCRIPTION
**Pull request by hickscorp**
Also tags a v1.0-rc1.

### CODE REVIEW

**Comment by hickscorp** [comment]
@afmfe-iul Transfer credit problem should also be fixed.

File: deploy/90_seedDevEnv.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { Exchange, Fast } from '../typechain';
 import { toBaseUnit, ZERO_ADDRESS } from '../src/utils';
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  return;
```
Comment (jamesduncombe) [inline deploy/90_seedDevEnv.ts:9]: Do we still want this?

File: deploy/80_seedStaging.ts [hunk 2]
```diff
@@ -3,17 +3,14 @@ import { DeployFunction } from 'hardhat-deploy/types';
 import { ethers, getNamedAccounts } from 'hardhat';
 import { deployFast, fastMint } from '../tasks/fast';
 import { Exchange, Fast } from '../typechain';
-import { toBaseUnit, ZERO_ADDRESS } from '../src/utils';
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  return;
```
Comment (jamesduncombe) [inline deploy/80_seedStaging.ts:8]: Do we still want this?

Comment (hickscorp) [reply deploy/80_seedStaging.ts:8]: Probably not.

File: deploy/90_seedDevEnv.ts (hunk 1 above)
Comment (hickscorp) [reply deploy/90_seedDevEnv.ts:9]: We really don't :D

**Review by jamesduncombe — APPROVED** [review]

**Pull request merged**